models/public/yolox-tiny/model.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from torch import load
+import torch.nn as nn
+from models.yolox import YOLOX
+from models.yolo_pafpn import YOLOPAFPN
+from models.yolo_head import YOLOXHead
+from models.network_blocks import SiLU
+
+
+def create_model(weights):
+    in_channels = [256, 512, 1024]
+    num_classes = 80
+    depth = 0.33
+    width = 0.375
+    backbone = YOLOPAFPN(depth, width, in_channels=in_channels)
+    head = YOLOXHead(num_classes, width, in_channels=in_channels)
+    model = YOLOX(backbone, head)
+
+    model.apply(init_yolo)
+    model.head.initialize_biases(1e-2)
```

Comment:
do we really need weights initialization if you load checkpoint?

Reply:
Done

Reply:
not all, model.apply is also weights initialization

Reply:
If model is converted without model.apply, we get a decrease in accuracy of about 0.5-1.1 % depending on the metric.